src/inverseRobot.js

```diff
@@ -25,6 +25,23 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const arrValues = Object.values(robot);
+  const uniqueArr = new Set(arrValues);
+  const newObj = {};
+
+  if (arrValues.length !== uniqueArr.size) {
+    return null;
+  }
+
+  if (!Object.keys(robot).length) {
+    return newObj;
+  }
+
+  for (const key in robot) {
+    newObj[`${robot[key]}`] = `${key}`;
```

Comment:
```suggestion
    newObj[robot[key]] = key;
```

do you really need template strings here?